docs/REDUX_ACTIONS.md

```diff
@@ -0,0 +1,107 @@
+# Actions flow from user perspective
+## Views action execution
+
+## Dashboard
+![alt text](assets/dashboard_screen.png "Dashboard Screen")
+
+When entering actions execution:
+
+| Module          |                    Function                 |
+| --------------- |---------------------------------------------|
+| Latest activity | `loadTransactions()`                        |
+| News            | `liskServiceApi.getNewsFeed()`              |
+| Price chart     | `liskServiceApi.getCurrencyGraphData(step)` |
+
+
+## Wallet
+![alt text](assets/wallet_screen.png "Wallet Screen")
+
+When entering actions execution order:
+
+1. `TRANSACTION_FILTERED`
+2. `ADD_FILTER`
+3. `TRANSACTION_LOADED`
+4. `ACCOUNT_ADD_VOTES` for *Account Info*
+
+On every tab change it executes `ADD_FILTER` action and reloading transactions except *Account Info*.
+
+## Delegates
+![alt text](assets/delegates_screen.png "Delegates Screen")
+
+
+When entering actions execution order:
+
+1. `VOTE_LOOKUP_STATUS_CLEARED`
+2. `DELEGATES_ADDED`
+3. `VOTES_ADDED`
+4. `VOTES_UPDATED`
+
+## User scenarios action execution
+
+## Login
+![alt text](assets/login_screen.png "Login Screen")
+
+After clicking `Log in` it executes:
+- `activePeerSet` with passphrase
+  ↪️ Login Middleware
+  &nbsp;&nbsp;&nbsp;&nbsp;⬇️ `getAccounts`
+  &nbsp;&nbsp;&nbsp;&nbsp;↪️ Socket Middleware
+  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;⬇️ `socketSetup`
+
+  ↪️ FollowedAccount Middleware
+  &nbsp;&nbsp;&nbsp;&nbsp;⬇️ `followedAccountFetchedAndUpdated` for each followed account
+
+
+## Logout
+![alt text](assets/delegates_screen.png "Logout Screen")
+
+After clicking `Logout` it executes:
+
+- `accountLoggedOut`
+  ↪️ Socket Middleware
+  &nbsp;&nbsp;&nbsp;&nbsp;⬇️ `closeConnection`
+
+
+### Sending
+![alt text](assets/send_screen.png "Sending Screen")
+
+
+After clicking `Send` it executes:
+- Calls `activePeer.transactions.broadcast(transaction)` and then dispatches `TRANSACTION_ADDED`
+
+
+### Delegates voting
+![alt text](assets/voting_confirm_screen.png "Voting Screen")
+
+
+After clicking `Confirm` it executes:
+- Calls `activePeer.transactions.broadcast(transaction)` and then dispatches `TRANSACTION_ADDED`
```

Comment:
Delegates voting section should also cover what actions are dispatched when selecting/deselecting delegates from the list.